src/components/RegisterForm.js

```diff
@@ -143,6 +143,7 @@ const RegisterForm = (props) => {
     }
     
     const isValidEmail = (emailValue) => {
+        // eslint-disable-next-line no-useless-escape
```

Comment:
Learn about this rule, it is a valid concern.

Reply:
Done